docs/advanced_usage/how_to_auxiliary.rst

```diff
@@ -279,3 +279,40 @@ More examples are available under :py:mod:`pykiso.lib.auxiliaries`.
     of threading, only the base class needs to be changed from
     ``AuxiliaryInterface`` to ``MpAuxiliaryInterface``. The actual
     implementation does not need any adaptation.
+
+.. _aux_without_connector:
+
+Auxiliary without connector
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If by default all auxiliaries require a connector, it is possible to create
+one without a connector. To do this, an attribute connector_required will have
+to be set to False when the auxiliary is initialized.
+Then, no connector option should be specified in the yaml configuration file.
```

Comment:
```suggestion
If by default all auxiliaries require a connector, in some specific cases, 
it may complicate the total implementation. Therefor `connector_required` 
parameter was defined.

.. notes:: Auxiliaries that should fall into this category will need to be discussed case by case.

..warning:: Auxiliaries entering this category will raise an error if a connector is indeed assigned to it in the .yaml. Hybrid cases do not exist.
```

Reply:
Done